bin/convert-argv.js

```diff
@@ -18,7 +18,7 @@ module.exports = function(yargs, argv, convertOptions) {
 	}
 	if(argv.p) {
 		argv["optimize-minimize"] = true;
-		argv["define"] = [].concat(argv["define"] || []).concat("process.env.NODE_ENV=\"production\"");
+		argv["define"] = Array.prototype.concat(argv["define"] || []).concat("process.env.NODE_ENV=\"production\"");
```

Comment:
Ok technically it's correct, so I did some [measurements](https://gist.github.com/sokra/25c78d9b26f158900821d9159099de16) about the performance. Here are the results:

``` text
node 8.1.4
Array.prototype.concat: 1527.249ms
Array.prototype.concat.call: 411.567ms
[].concat: 454.620ms
Array.prototype.concat.call cached: 419.850ms

node 6.11.1
Array.prototype.concat: 1097.169ms
Array.prototype.concat.call: 654.220ms
[].concat: 820.285ms
Array.prototype.concat.call cached: 664.872ms
```

=> Use Array.prototype.concat.call